flink-core/src/main/java/org/apache/flink/configuration/GlobalConfiguration.java

```diff
@@ -44,7 +44,8 @@
 	public static final String FLINK_CONF_FILENAME = "flink-conf.yaml";
 
 	// the keys whose values should be hidden
-	private static final String[] SENSITIVE_KEYS = new String[] {"password", "secret", "fs.azure.account.key"};
+	private static final String[] SENSITIVE_KEYS = new String[] {"password", "secret",
+		"fs.azure.account.key", "metrics.reporter.dghttp.apikey"};
```

Comment:
This is too specific. The `dhtp` infix is arbitrary; users are free to use any value here.
`apikey` on it's own should be sufficient.

Reply:
My idea was to follow the previous example when the Azure key was added and for that the full key was used. 
But I agree with your opinion so I will fix it shortly.